webapp/components/Modal/DeleteModal.vue

```diff
@@ -33,6 +30,7 @@ export default {
   props: {
     name: { type: String, default: '' },
     type: { type: String, required: true },
+    callbacks: { type: Object, required: true },
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-03T12:58:07Z" title="Monday, June 3rd 2019, 2:58:07 pm +02:00">Jun 3, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -33,6 +30,7 @@ export default {
   props: {
     name: { type: String, default: '' },
     type: { type: String, required: true },
+    callbacks: { type: Object, required: true },
```


If you assign each callback a prop, you could make it optional and have `() => {}` as the default.